### DISCUSSION TITLE
3.2.1-release master-server启动报错

### DISCUSSION THREAD

**Discussion by Carl7713** [apache/dolphinscheduler#15673] [General]
3.2.1-release 拉下来怎么不能直接部署 run起来
配置信息加载不到
No qualifying bean of type 'org.apache.dolphinscheduler.registry.api.Registry' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {}

**Comment by HaxMing** [comment]
请问解决了么

**Comment by versechen** [comment]
相同报错。
背景：dockerdesktop k8s集群安装。
![image](https://github.com/user-attachments/assets/57461d22-fbe4-49b2-9653-fa5557818f48)

[WI-0][TI-0] - 2024-11-06 09:26:47.667 WARN  [Worker-Server] o.s.b.w.s.c.AnnotationConfigServletWebServerApplicationContext:[591] - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'taskInstanceOperatorImpl': Unsatisfied dependency expressed through field 'taskInstanceOperationFunctionManager'; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'taskInstanceOperationFunctionManager' defined in URL [jar:file:/opt/dolphinscheduler/libs/dolphinscheduler-worker-dev-SNAPSHOT.jar!/org/apache/dolphinscheduler/server/worker/runner/operator/TaskInstanceOperationFunctionManager.class]: Unsatisfied dependency expressed through constructor parameter 2; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'taskInstanceDispatchOperationFunction' defined in URL [jar:file:/opt/dolphinscheduler/libs/dolphinscheduler-worker-dev-SNAPSHOT.jar!/org/apache/dolphinscheduler/server/worker/runner/operator/TaskInstanceDispatchOperationFunction.class]: Unsatisfied dependency expressed through constructor parameter 1; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'workerTaskExecutorFactoryBuilder' defined in URL [jar:file:/opt/dolphinscheduler/libs/dolphinscheduler-worker-dev-SNAPSHOT.jar!/org/apache/dolphinscheduler/server/worker/runner/WorkerTaskExecutorFactoryBuilder.class]: Unsatisfied dependency expressed through constructor parameter 3; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type 'org.apache.dolphinscheduler.plugin.storage.api.StorageOperator' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {}
